tools/commentMonitor/Dockerfile

```diff
@@ -4,18 +4,11 @@ FROM golang:1.12-alpine AS builder
 RUN apk update && apk add --no-cache git \
     ca-certificates
 
-ENV GO111MODULE=on
-
 WORKDIR /app
-COPY tools/commentMonitor/main.go .
-COPY tools/commentMonitor/client.go .
-COPY tools/commentMonitor/ghclient.go .
-
-COPY go.mod .
-COPY go.sum .
-RUN go mod download
+COPY tools/commentMonitor/*.go ./
+COPY go.* ./
 
-RUN CGO_ENABLED=0 GOOS=linux go build -o /go/bin/commentMonitor
+RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux go build -o /go/bin/commentMonitor
```

Comment:
`CGO_ENABLED=0` can be replaced with `-tags netgo` to keep things consistent among the Dockerfiles, It should work the same for this.